org.eclipse.tm4e.core/META-INF/MANIFEST.MF

```diff
@@ -8,7 +8,10 @@ Bundle-Version: 0.1.0.qualifier
 Require-Bundle: org.apache.batik.css;bundle-version="1.7.0";resolution:=optional,
  org.apache.batik.util;bundle-version="1.7.0";resolution:=optional,
  com.google.gson;resolution:=optional,
- org.eclipse.core.runtime
+ org.eclipse.core.runtime,
+ org.jcodings;bundle-version="1.0.18",
+ org.joni;bundle-version="2.1.11",
+ org.junit;bundle-version="4.11.0"
```

Comment:
Not necessarily, I'll move the declaration to junit dependency back into the pom.xml. Thanks for catching!